spec/controllers/invoices/invoices_items_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
```

Comment:
I only created this spec, but was having a brain-fart trying to figure out how to write the tests, so I just skipped them and left a comment as a reminder to come back to this.